[CLOUD-256] [Doc] Upgrade from 0.3.x to 0.4.0 (#528)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/operations-manual.adoc

==== Execution

> To upgrade the _Stratio Cloud Provisioner_ version from 0.3 to 0.4 you must run the script _upgrade-provisioner++_++.py_ from the workspace where the cluster was installed.

Reviewer: ```suggestion
To upgrade the _Stratio Cloud Provisioner_ version from 0.3 to 0.4 you must run the script _upgrade-provisioner++_++.py_ from the workspace with which the cluster was installed.
```

You can refer to the script help with the following command:

[source,bash]
----
python3 upgrade-provisioner.py -h
----

Example:

[source,bash]
----
python3 upgrade-provisioner.py -p <vault_pass>
----

> The _cluster.yaml_ and _secrets.yml_ descriptors must be present in the workspace. If the _cluster.yaml_ descriptor has a different name or is located in a different path, it can be specified using the '-d' parameter.

Reviewer: ```suggestion
The _cluster.yaml_ and _secrets.yml_ descriptors must be present in the workspace. If the _cluster.yaml_ descriptor has another name or is in another path, it can be specified with the `-d` parameter.
```
